yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -485,3 +485,33 @@ func TestSingleXMLIssueIsDetected(t *testing.T) {
 	fatalIfError(t, err)
 	assert.Equal(t, 1, len(issues))
 }
+
+func TestRegexFunctionsIssue(t *testing.T) {
+	dmlStmts := []string{
+		`SELECT regexp_count('This is an example. Another example. Example is a common word.', 'example')`,
+		`SELECT regexp_instr('This is an example. Another example. Example is a common word.', 'example')`,
+		`SELECT regexp_like('This is an example. Another example. Example is a common word.', 'example')`,
+		`SELECT regexp_count('abc','abc'), regexp_instr('abc','abc'), regexp_like('abc','abc')`,
```

Comment:
should we add a test to the `issues_dml_test.go` ?

Reply:
Ah, missed it, thanks!